### ISSUE TITLE
v13: PNG images with `?format=webp` are much larger than in v10

### ISSUE THREAD

**Issue by PerplexDaniel** [umbraco/Umbraco-CMS#16768]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

13.4.0

### Bug summary

PNG images converted to WebP with ImageSharp using `?format=webp` in Umbraco v13 are much bigger than the exact same ones in Umbraco v10. This does not seem to happen with JPGs but I have not tested that extensively.

This has been reported on the [ImageSharp GitHub](https://github.com/SixLabors/ImageSharp/discussions/2766) and the conclusion is that ImageSharp v3+ defaults to `Lossless` encoding rather than `Lossy` like in v2.

For a sample 9MB image this means the resulting WebP image is not 400 KB, but 7.5 MB instead.

In order to fix this, Umbraco should configure the WebP encoder to default to Lossy instead of Lossless so it behaves like in Umbraco v10. I made a PR here: #16769.

Default Umbraco 13 (Lossless):
![image](https://github.com/umbraco/Umbraco-CMS/assets/24297947/9e11dc98-b778-45eb-bf77-8139eba368f2)

Umbraco 13 with Lossy:
![image](https://github.com/umbraco/Umbraco-CMS/assets/24297947/1f2682ba-441f-4e47-9454-6623000de4f1)

### Steps to reproduce

Use this file:
<details>
<summary>earth.png</summary>
<img src="https://github.com/umbraco/Umbraco-CMS/assets/24297947/ac1bd3a0-9b9d-44a3-86bb-0be3edd6ea45"/>
</details>

1. Install a new Umbraco v13 site (`dotnet new umbraco -n webp`).
2. Place `earth.png` in the `wwwroot/media` folder (no need to upload in the media section itself)
4. Navigate to `https://localhost:<PORT>/media/earth.png?format=webp` in your browser
5. Observe the file is ~7.5 MB:
![image](https://github.com/umbraco/Umbraco-CMS/assets/24297947/4771ae78-8779-4463-b6ea-b65355a35cf1)

The fix is to configure ImageSharp to default to Lossy encoding for WebP:

1. Add to Program.cs -- right above `WebApplication app = builder.Build();`
```csharp
builder.Services.Configure<ImageSharpMiddlewareOptions>(config =>
{
    config.Configuration.ImageFormatsManager.SetEncoder(WebpFormat.Instance, new WebpEncoder
    {
        // Lossy encoding creates files that are 10x smaller than Lossless (the default).
        FileFormat = WebpFileFormatType.Lossy
    });
});
```
2. **!IMPORTANT!** Empty the media cache directory `umbraco/Data/TEMP/MediaCache`
3. Navigate to `https://localhost:<PORT>/media/earth.png?format=webp` in your browser again
4. Observe the file is now only 397 kB:
![image](https://github.com/umbraco/Umbraco-CMS/assets/24297947/89004e6d-d474-4ff8-b7ff-7d1bca5798ed)

**Comment by elit0451** [comment]
Seems like this has been fixed in #16769, so I will go ahead and close it 🙂

**Comment by PerplexDaniel** [comment]
For future readers confused why this was removed from release v13.5.0, per [Umbraco reply on the PR](https://github.com/umbraco/Umbraco-CMS/pull/16769#issuecomment-2270586367) this was somehow considered a breaking change so it will be postponed until v15 instead.

As such the first LTS to get it will be v17 😞. Very surprising/sad, especially since it was working fine in v10 so this regression is just kept in v13 LTS for no good reason. Image sizes have a big impact on page speed and user experience but this is clearly not important to Umbraco.

If you use PNG images and `?format=webp` on your v13 website you should not wait for Umbraco and just add this to Program.cs -- right above `WebApplication app = builder.Build();`:

```csharp
builder.Services.Configure<ImageSharpMiddlewareOptions>(config =>
{
    config.Configuration.ImageFormatsManager.SetEncoder(WebpFormat.Instance, new WebpEncoder
    {
        // Lossy encoding creates files that are 10x smaller than Lossless (the default).
        FileFormat = WebpFileFormatType.Lossy
    });
});
```
